tools/src/test/java/org/apache/kafka/tools/consumer/ConsoleConsumerTest.java

```diff
@@ -49,14 +87,25 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
+@ExtendWith(value = ClusterTestExtensions.class)
+@Tag("integration")
 public class ConsoleConsumerTest {
 
+    private final ClusterInstance cluster;
+    private final String topic = "test-topic";
+    private final String transactionId = "transactional-id";
+    private final ObjectMapper objectMapper = new ObjectMapper();
+
+    public ConsoleConsumerTest(ClusterInstance cluster) {
```

Comment:
not all test cases use `ClusterTest`, so you can't define the `ClusterInstance` in constructor